### DISCUSSION TITLE
What determines which wallets can connect only with mobile qr scanning vs the browser option

### DISCUSSION THREAD

**Discussion by kmadk** [WalletConnect/walletconnect-monorepo#3956] [Developer Support] [answered]
Basically title, but for example when connecting xdefi you have the option to connect through mobile qr scan or browser, but leap for example only mobile qr. Is there any way to change this and why is this.

**Comment by glitch-txs** [comment, marked as answer]
Hi, this information is taken from the Cloud explorer, and it's given by each wallet when they submit a form, you can also create custom configurations if you need to.

**Reply by kmadk** [reply]
how difficult is the custom config for something like LEAP browser. i assume there is a guide in the github about how to make custom configs. Thanks so much for the help thus far!
